solvation_analysis/plotting.py

```diff
@@ -69,10 +73,12 @@ def plot_shell_size_histogram(solution):
     fig : Plotly.Figure
 
     """
-    speciation_data = solution.speciation.speciation_data
-    # I suggest using the groupby operation
-    # df.groupby('solute_index').sum()
-    # TODO:
+    speciation_data = solution.speciation.speciation_data # DataFrame
+    bn = speciation_data.groupby("bn").sum()
```

Comment:
Is there a reason to hardcode "bn", "fec" and so forth here? What if the system in question doesn't contain these residues?

Reply:
Hello! Thank you for catching this. I've changed the code so that doesn't hardcode the species names. I'll be pushing new changes to the code shortly!